src/components/CreateResetPassword/CreateResetPassword.js

```diff
@@ -46,12 +53,15 @@ class CreateResetPassword extends Component {
   constructor(props) {
     super(props);
 
-    this.translateNamespace = 'stripes-core.createResetPassword';
-    this.passwordMatchError = {
-      code: 'password.match.error',
-      type: 'error',
+    this.translationNamespaces = {
+      module: 'stripes-core',
+      smartComponents: 'stripes-smart-components',
+      page: 'stripes-core.createResetPassword',
+      errors: 'stripes-core.errors',
+      button: 'stripes-core.button',
     };
-    this.newPassword = React.createRef();
+    this.newPasswordField = props.stripes.connect(PasswordValidationField);
+    this.passwordMatchErrorCode = 'password.match.error';
     this.state = {
```

Comment:
Let's have `state` right after the 

`super(props)` call. So it would be easier to find what is a component state.